projects/com.oracle.truffle.llvm.parser/src/com/oracle/truffle/llvm/parser/util/LLVMBitcodeTypeHelperInterface.java

```diff
@@ -0,0 +1,17 @@
+package com.oracle.truffle.llvm.parser.util;
+
+import uk.ac.man.cs.llvm.ir.types.Type;
+
+public interface LLVMBitcodeTypeHelperInterface {
```

Comment:
Can we rename this class to `LLVMBitcodeTypeHelper` and its default implementation to `LLVMBitcodeTypeHelperImpl`?